### ISSUE TITLE
Pumpkin quest

### ISSUE THREAD

**Issue by Kuperus** [Storkman101/Void-World#41]
some of the pumpkin faces do not match the ones that are asked... ( right now it`s 1 that is wrong on the left side, not sure which)

**Comment by Storkman101** [comment]
Jup I know some of the pumpkins quests are still bugged right now. Still have to dive into that. But I forgot some of the hidden places of the pumpkins so yeah. I have to find them myself.

**Comment by Kuperus** [comment]
actually you don`t have to find them, philosopher stone gives you workaround in combination with chisel...

**Comment by Storkman101** [comment]
Yeah I know, still have to fix that. I dont mind them being chisable due to the bug but it is annoying melons can be transmuted into pumpkins. It is difficult to disable one of them or even both.

**Comment by Kuperus** [comment]
maybe better or easier to disable the chisel for pumpkins
